### PR TITLE
Rename executeTransaction to executeTx

### PR DESCRIPTION
**Pull request by thdailong**
Closes #x

## Proposed Changes

+ Rename all the file executeTransaction to executeTx
+
+

### CODE REVIEW

**Comment by sczembor** [comment]
There are some tests that didn't pass:
```Error: ➤ YN0000: scripts/withdraw/common.ts(1,10): error TS2440: Import declaration conflicts with local declaration of 'executeTx'.
```

**Comment by thdailong** [comment]
> There are some tests that didn't pass: `Error: ➤ YN0000: scripts/withdraw/common.ts(1,10): error TS2440: Import declaration conflicts with local declaration of 'executeTx'.`

Yes I have seen it. The executeTx function also being export but in the file we have import executeTx. This make the conflicts and I think I will change the export function

**Review by rmeena840 — APPROVED** [review]

File: examples/htlc-pyteal-ts/scripts/withdraw/common.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export async function executeTx(
 	txnParams: wtypes.ExecParams
```
Comment (vuvth) [inline examples/htlc-pyteal-ts/scripts/withdraw/common.ts:25]: You import `executeTx` from `algob/web` but at this files you defined another `executeTx`. The name is same and you got build problem. This is the reason your code can't build. Please fix this problem. @thdailong.

File: packages/web/src/lib/wallectconnect-mode.ts [hunk 2]
```diff
@@ -217,7 +217,7 @@ export class WallectConnectSession {
 	 * Execute single transaction or group of transactions (atomic transaction)
 	 * @param execParams transaction parameters or atomic transaction parameters
 	 */
-	async executeTransaction(
```
Comment (vuvth) [inline packages/web/src/lib/wallectconnect-mode.ts:220]: Let follow @robert-zaremba guide
```typescript
@deprecated
executeTransaction(....) {
  return executeTx(...)
}
```
Useful resource: https://www.typescriptlang.org/play#example/jsdoc-deprecated

File: packages/web/src/lib/web-mode.ts [hunk 3]
```diff
@@ -130,7 +130,7 @@ export class WebMode {
 	 * Execute single transaction or group of transactions (atomic transaction)
 	 * @param execParams transaction parameters or atomic transaction parameters
 	 */
-	async executeTransaction(execParams: ExecParams | ExecParams[]): Promise<JsonPayload> {
```
Comment (vuvth) [inline packages/web/src/lib/web-mode.ts:133]: also use  `@deprecated` at here.

**Review by vuvth — COMMENTED** [review]
Thanks for your PR. I left with few suggestion.

**Pull request merged**